spec/models/miq_report_spec.rb

```diff
@@ -765,10 +770,18 @@
                             cpu_usagemhz_rate_average_low_over_time_period
                             derived_memory_used_high_over_time_period
                             derived_memory_used_low_over_time_period)}})
+        report.generate_table(:userid => "admin")
       end
-      let(:ems) { FactoryBot.create(:ems_vmware, :zone => @server.zone) }
 
-      it "runs report" do
+      it "runs report with polymorphic references" do
+        FactoryBot.create(:metric_rollup_vm_daily, :resource => vm, :time_profile => time_profile)
+
+        report = MiqReport.new(
+          :title     => "vim_perf_daily.yaml",
+          :db        => "VimPerformanceDaily",
+          :col_order => %w[timestamp cpu_usagemhz_rate_average max_derived_cpu_available
+                           resource.derived_memory_used_low_over_time_period]
+        )
```

Comment:
Are we testing that it runs without raising?  Should we be doing that or checking the results of generate_table?

Reply:
I didn't write this spec... @kbrock ?

Reply:
yea, this is testing that it doesn't puke.